docs/developer/apis/camino-node-apis/p-chain.md

```diff
@@ -938,6 +938,179 @@ curl -X POST --data '{
 }
 ```
 
+### platform&#46;getClaimables
+
+Returns claimable rewards
+
+:::caution Work In Progress
+
+This API method's documentation is **work-in-progress**. It will be updated as the implementation is ready on the Camino Network.
+
+:::
+
+**Signature**
+
+```
+platform.getClaimables(
+    {
+        locktime: uint64
+        threshold: uint32
+        addresses: []string
+        depositTxIDs: []string
+    }
+
+) -> {
+    depositRewards: uint64,
+    validatorRewards: uint64,
+    expiredDepositRewards: uint64,
+}
+```
+
+- `locktime`: locktime of the rewards
```

Comment:
locktime is locktime of avax control group that owns claimables (validator rewards, expired deposit rewards)

Reply:
Fixed